integrations/ScriptLoader.js

```diff
@@ -9,10 +10,18 @@ const ScriptLoader = (id, src, async = defaultAsyncState) => {
   js.async = async === undefined ? defaultAsyncState : async;
   js.type = "text/javascript";
   js.id = id;
-  const e = document.getElementsByTagName("script")[0];
-  logger.debug("==parent script==", e);
-  logger.debug("==adding script==", js);
-  e.parentNode.insertBefore(js, e);
+  const h = document.getElementsByTagName("head")[0];
+  if (Object.keys(h).length !== 0) {
+    const e = h.getElementsByTagName("script")[0];
```

Comment:
Can we not insert it before the first child of the head element? Does it have to be a script element?

Reply:
Updated.